src/execution.cc

```diff
@@ -202,15 +202,16 @@ void PrintToolTargetsList(const vector<Node*>& nodes, int depth, int indent) {
 
```

Comment:
I really wish we could just bite the bullet and use C++11.

9 years seems long enough to support legacy systems for a tool like this...